### PR TITLE
add app, display, button and buttonpanel components

### PR DESCRIPTION
**Pull request by grace-mugoiri**

### CODE REVIEW

**Comment by grace-mugoiri** [comment]
How can I fix these linters? i have the rules in the eslint config file, but they show

**Comment by shloch** [comment]
Hello @grace-mugoiri , Great job
But we have some ameliorations to do 

## CHANGES REQUIRE
Please for the structure of this project, is required to 
- place `APP` component directly  in the `src/components` directory

see more changes to do below :

File: src/components/display.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <div className='display'>
+      <div className='display-value'>
+        {value}
+      </div>
+    </div>
+  );
+}
+
+export default display;
```
Comment (shloch) [inline src/components/display.js]: - The result prop should be a String.
- The default value of the result prop should be 0.

Try to do some research regarding `proptypes` and `defaultProps `
and solve these two issues

File: src/components/button.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <div>
+      <h1>
+        {props.name}
+      </h1>
+    </div>
+  );
+}
+
+export default button;
```
Comment (shloch) [inline src/components/button.js]: - The button name prop should be a String.

`Button.propTypes = {
  name: PropTypes.string
};`

**Comment by grace-mugoiri** [comment]
Please review the code, fixed errors as per reviewers/project requirements.

**Pull request merged**